src/events/index.js

```diff
@@ -111,6 +111,41 @@ function fetchExchangeLogs(eventName, fromBlock, toBlock) {
   return fetchLogs(SWAP_LEGACY_CONTRACT_ADDRESS, abis[SWAP_LEGACY_CONTRACT_ADDRESS], topic, fromBlock, toBlock)
 }
 
+const swapLegacyContractCreationBlock = `0x${(4349701).toString(16)}`
+
+function fetchFilledExchangeLogsForMakerAddress(makerAddress) {
+  const abiInterface = new ethers.utils.Interface(abis[SWAP_LEGACY_CONTRACT_ADDRESS])
+  const topics = abiInterface.events.Filled.encodeTopics([makerAddress.toLowerCase()])
+  return fetchLogs(
+    SWAP_LEGACY_CONTRACT_ADDRESS,
+    abis[SWAP_LEGACY_CONTRACT_ADDRESS],
+    topics,
+    swapLegacyContractCreationBlock,
+  )
+}
+
+function fetchCanceledExchangeLogsForMakerAddress(makerAddress) {
+  const abiInterface = new ethers.utils.Interface(abis[SWAP_LEGACY_CONTRACT_ADDRESS])
+  const topics = abiInterface.events.Canceled.encodeTopics([makerAddress.toLowerCase()])
+  return fetchLogs(
+    SWAP_LEGACY_CONTRACT_ADDRESS,
+    abis[SWAP_LEGACY_CONTRACT_ADDRESS],
+    topics,
+    swapLegacyContractCreationBlock,
+  )
+}
+
+function fetchFailedExchangeLogsForMakerAddress(makerAddress) {
+  const abiInterface = new ethers.utils.Interface(abis[SWAP_LEGACY_CONTRACT_ADDRESS])
+  const topics = abiInterface.events.Failed.encodeTopics([null, makerAddress.toLowerCase()])
```

Comment:
just want to make sure this is correct. `[null, makerAddress.toLowerCase()]` whereas the previous two just pass the makerAddress only

Reply:
Yep it is, `makerAddress` is the second param for this event, first for the other two